docs/state/preredux/README.md

```diff
@@ -0,0 +1,62 @@
+## Tidepool uploader state pre-redux
+
+This document outlines the application state management in the current (v0.233.0) version of the Tidepool uploader, prior to migrating the application state management into [redux](http://redux.js.org/).
+
+The Tidepool uploader currently manages state using a single global object to represent the entirety of the application's state. All possible changes to the global state are performed through a set of functions found in [`lib/state/appActions.js`](https://github.com/tidepool-org/chrome-uploader/blob/master/lib/state/appActions.js).
+
+Simple attributes of the global state object:
+
+- `dropMenu` - a boolean encoding whether the dropdown menu available by clicking on the logged-in user's name in the upper-right corner is currently visible (`true`) or hidden (`false`)
+
+- `howToUpdateKBLink` - the URL for [the Tidepool knowledge base article explaining how to update the uploader](https://tidepool-project.helpscoutdocs.com/article/6-how-to-install-or-upgrade-the-tidepool-uploader-gen)
+
+- `page` - the current "page" of the app being rendered
+   + "page" in scare quotes because of the uploader's nature as a single-page app with no router
```

Comment:
Hahaha, former linguist in me comes out. Wikipedia says it best: [scare quotes](https://en.wikipedia.org/wiki/Scare_quotes). Here the point is that calling it a "page" is not very literal since we don't have a router in the uploader.